doc/examples/widgets.md

```diff
@@ -0,0 +1,28 @@
+# Widgets
+
+One cool feature when developing games with Flutter is the ability to use Flutter's extensive toolset for building UIs, Flame tries to expand that introducing widgets which are specially made with games in mind.
+
+Here you can find all the available widgets provided by Flame.
+
+You can also see all the widgets showcased inside a Dashbook sandbox [here](/doc/examples/widgets)
+
+## Nine Tile Box
+
+A Nine Tile Box is a rectangle drawn using a grid sprite.
+
+The grid sprite is a 3x3 grid and with 9 blocks, representing the 4 courners, the 4 sides and the middle.
+
+The corners are drawn at the same size, the sides are streched on the side direction and the middle is expanded both ways.
```

Comment:
Please link to the in-game version here and briefly explain the difference

Reply:
Done, let me know if it is ok

Reply:
it's good!